vaadin-spring/src/main/resources/META-INF/services/javax.annotation.processing.Processor

```diff
@@ -0,0 +1 @@
+com.vaadin.flow.spring.ConfigurationPropertiesAnnotationProcessor
```

Comment:
new line

Reply:
mmm... it seems I lost my style configuration. I'll fix all the new lines you commented. Thanks